### PR TITLE
[web-5335] support for translating markdown shortcodes

### PR DESCRIPTION
**Pull request by bgdeutsch**
### What does this PR do? What is the motivation?
- rename all shortcodes to end in `.en.md`.   this is needed to fix hugo's shortcode lookup (it will never render the english shortcode if it doesn't have `.en.` in the filename and a translation exists)
- downloads completed shortcode translations from Transifex
- update `translate.yaml` to support new file names

### Merge instructions
- [x] Please merge after websites reviews

Spot check various pages in the preview site and confirm shortcodes are rendering correctly including translated pages.

Example:
https://docs-staging.datadoghq.com/brian.deutsch/web-5335-translate-shortcodes/tracing/services/resource_page/#out-of-the-box-graphs
- JA should render translated version
- KO should render english version since no KO translation exists

### CODE REVIEW

**Comment by hestonhoffman** [comment]
I took a look at a selection of these and they're looking good

**Review by hestonhoffman — APPROVED** [review]

**Review by hestonhoffman — APPROVED** [review]

File: layouts/shortcodes/apm-ssi-uninstall-linux.en.md
Comment (StefonSimmons) [inline layouts/shortcodes/apm-ssi-uninstall-linux.en.md:1]: @bgdeutsch  non-blocking but this doesnt have an equivalent `/ja` translation

**Review by StefonSimmons — APPROVED** [review]
lgtm!

File: layouts/shortcodes/apm-ssi-uninstall-linux.en.md
Comment (bgdeutsch) [reply layouts/shortcodes/apm-ssi-uninstall-linux.en.md:1]: @StefonSimmons yeah not all of the shortcodes have translations yet, in this PR i've downloaded whichever ones were 100% reviewed from Transifex.

**Pull request merged**